### PR TITLE
Added dataset for Places365 small --Issue 288

### PR DESCRIPTION
**Pull request by SoumyadeepJana**
@cyfra @Conchylicultor  @rsepassi There were no labels for test and validation data,how to deal with this?
I have included 'test' as a label for all the images in test and validation sets.
https://github.com/tensorflow/datasets/issues/288

### CODE REVIEW

**Comment by us** [comment]
@SoumyadeepJana your commits are included the ham1000 datasets commit. You should create different branch to all different pr to avoid the conflicts.

**Comment by SoumyadeepJana** [comment]
https://github.com/tensorflow/datasets/issues/288
